### PR TITLE
Added light-up-Chinatown progressBar fetchData functions

### PR DESCRIPTION
**Pull request by FelicityZhang**

### CODE REVIEW

**Comment by FelicityZhang** [comment]
Screen Shot 2020-11-13 at 10 56 44 PM

File: src/components/LightUpChinatown/LightUpChinatownPage.tsx [hunk 1]
```diff
@@ -12,22 +12,23 @@ import DonationSection from './DonationSection';
...
+  };
 
   useEffect(() => {
-    setContributions(15000);
-    // TODO: use the below after backend route to fetch raised is implemented
-    //
-    // const fetchData = async () => {
-    // setContributions(data.contributions.raised);
-    // };
+    fetchData(1);
```
Comment (justintmckibben) [inline src/components/LightUpChinatown/LightUpChinatownPage.tsx:31]: is this created in prod yet?

**Review by justintmckibben — APPROVED** [review]

File: src/components/LightUpChinatown/LightUpChinatownPage.tsx (hunk 1 above)
Comment (FelicityZhang) [reply src/components/LightUpChinatown/LightUpChinatownPage.tsx:31]: what does "prod" mean?
I followed by this:
Screen Shot 2020-11-13 at 11 51 59 PM
Is it correct?

Comment (justintmckibben) [reply src/components/LightUpChinatown/LightUpChinatownPage.tsx:31]: Mind adding it as a constant and importing it? `LIGHT_UP_CHINATOWN_PROJECT_ID`

Comment (FelicityZhang) [reply src/components/LightUpChinatown/LightUpChinatownPage.tsx:31]: Okay!

**Pull request merged**